<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tribler Aggregated Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="search_input">
         <property name="placeholderText">
          <string>Search query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="num_search_results_label">
      <property name="styleSheet">
       <string notr="true">margin-left: 10px;</string>
      </property>
      <property name="text">
       <string>0 search results</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="search_results_tree">
      <attribute name="headerDefaultSectionSize">
       <number>550</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Items</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
